docs/docs/policies/index.md

```diff
@@ -0,0 +1,66 @@
+# Getting Started with Policies
+
+Tracee's policies are how you describe what tracee should collect and what actions it should take.
+
+For example, we would like to collect all `sched_process_exec` events from the whole host:
```

Comment:
Instead of "we" and similar pronouns, try to express it more generic ☺️ 
"For example, the following policy collects all `sched_process_exec` events from the whole host:"

Reply:
well given you continue with "we" let's keep it:
"For example, the following policy we specify to collect all `sched_process_exec` events from the whole host:"